src/stringify.js

```diff
@@ -104,10 +119,10 @@ var BASE_16 = 16;
         escapable.lastIndex = 0;
         return escapable.test(string) ? '"' + string.replace(escapable, function (a)
         {
-            var c = meta[a];
-            return typeof c === 'string' ?
+            var c = SUBSTITUTION_TABLE[a];
+            return typeof c === constnts.STRING_TYPE ?
```

Comment:
constants

Reply:
btw, didn't this break anything?

Reply:
you're right. Let me write a failing test.